apps/server/src/connect/connectApi.ts

```diff
@@ -276,14 +277,16 @@ export class ConnectApi extends AggregatorAdapterBase {
   }
 
   async loadPopularInstitutions() {
+    const config = getConfig();
     this.context.updated = true;
     this.context.aggregator = null;
 
-    const recommendedInstitutions = await getRecommendedInstitutions(
-      this.context.job_type as MappedJobTypes,
-    );
+    const recommendedInstitutions = await getRecommendedInstitutions({
```

Comment:
This is more complex than it needs to be.

getRecommendedInstitutions could get the config itself and check for the prod env. This would mean less chance for integration failure and less tests.